laravel/app/Services/LoggingService/File/LoggingService.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Services\LoggingService\File;
+
+use App\Services\ConfigLoaderService\ConfigLoaderServiceException;
+use App\Services\ConfigLoaderService\IConfigLoaderService;
+use App\Services\LoggingService\ILoggingService;
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+
+/**
+ * Class LoggingService
+ *
+ * File logging service
+ *
+ * @package App\Services\LoggingService\File
+ */
+class LoggingService implements ILoggingService
+{
+    /** @var Logger $logger */
+    private $logger;
+
+    /** @var IConfigLoaderService $configLoaderService */
+    private $configLoaderService;
+
+    /** Path to use for logging when none found in config settings */
+    const DEFAULT_PATH = 'storage/logs/default.log';
+
+    /**
+     * LoggingService constructor.
+     *
+     * @param IConfigLoaderService $configLoaderService
+     */
+    public function __construct(IConfigLoaderService $configLoaderService)
```

Comment:
This is an example of dependency injection, pass the interface into the constructor of the function- and the DI will inject the appropriate concrete implementation as defined earlier. You would generally then store the object in a private variable for use within the class.